js/inputbar.js

```diff
@@ -76,6 +79,11 @@ weechat.directive('inputBar', function() {
             };
 
             $scope.completeNick = function() {
+                if ( $scope.command.startsWith('/') ) {
```

Comment:
This breaks nick completion in situations like `/query foo<tab>` for weechat <= 2.8, we should probably check for weechat >= 2.9 as well before aborting. But we know the weechat version anyway (`models.version[0] >= 2 && models.version[1] >= 9`).

Reply:
(the above is wrong, I committed a fixed version)